modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc

```diff
@@ -38,6 +38,16 @@ The contents of a concept module give the user descriptions and explanations nee
 * Avoid including action items. Action items belong in procedure modules. However, in some cases a concept or reference can include suggested actions when those actions are simple, are highly dependent on the context of the module, and have no place in any procedure module. In such cases, ensure that the heading of the concept or reference remains a noun phrase and not a gerund.
 
 
+////
+Include titles and alternative text descriptions for images.
+Alternative text should provide a textual, complete description of the image as a full sentence.
```

Comment:
The alternative text does not need to be a full sentence. It's okay if it is a sentence, but it should be as short and concise as possible to give a complete textual description of the image.

I think this is a helpful source for that from the w3. https://www.w3.org/WAI/tutorials/images/tips/

>The alt text should be the most concise description possible of the image’s purpose. If anything more than a short phrase or sentence is needed, it would be better to use one of the long description methods discussed in [complex images](https://www.w3.org/WAI/tutorials/images/complex/).